src/components/ticketsFlatList.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {View, StyleSheet, Text } from 'react-native';
+import { FlatList } from 'react-native-gesture-handler';
+import { JourneyOutput } from '../types/JourneyOutput';
+
+const styles = StyleSheet.create({
+  item: {
+    padding: 20,
+    marginVertical: 8,
+    marginHorizontal: 16,
+    backgroundColor: 'orange'
+  }
+});
+
+const listStyle = StyleSheet.create({
+  container: {
+    backgroundColor: '#83ccde',
+  },
+});
+
+type TicketFlatListProps = {
+  items: Array<JourneyOutput>;
+};
+
+const TicketsFlatList: React.FC<TicketFlatListProps> = ({ items }) => {
+  const renderItem = ({ item }) => (
+    <View style = {styles.item}>
+      <Text>{item.departureTime.slice(0,10)} {item.departureTime.slice(11,16)}</Text>
```

Comment:
I know what you're doing here, but most people wouldn't! It would be much better to pull out a helper method of the form:

```typescript
const mapIsoDateTimeToDisplayDateTime = (dateTime: string): string => {
   // do stuff
};
```

Really useful self-documentation as well as less repeated code.